Decred Wallet/Features/Custom Dialogs/AccountSelectorDialog/AccountSelectorDialog.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  AccountSelectorDialog.swift
+//  Decred Wallet
+//
+// Copyright (c) 2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+import UIKit
+import Dcrlibwallet
+
+typealias AccountSelectorDialogCallback = (_ selectedWalletId: Int, _ selectedAccount: DcrlibwalletAccount) -> Void
+
+class AccountSelectorDialog: UIViewController {
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var walletsTableView: UITableView!
+    @IBOutlet weak var walletsTableViewHeightConstraint: NSLayoutConstraint!
+    
+    private var dialogTitle: String!
+    private var callback: AccountSelectorDialogCallback!
+
+    var wallets = [Wallet]()
+    var selectedWallet: DcrlibwalletWallet?
+    var selectedAccount: DcrlibwalletAccount?
+
+    let accountCellRowHeight: CGFloat = 74
+    let walletHeaderSectionHeight: CGFloat = 36
+    
+    static func show(sender vc: UIViewController,
+                     title: String,
+                     selectedWallet: DcrlibwalletWallet?,
+                     selectedAccount: DcrlibwalletAccount?,
+                     callback: @escaping AccountSelectorDialogCallback) {
+
+        let dialog = AccountSelectorDialog.instantiate(from: .CustomDialogs)
+        dialog.dialogTitle = title
+        dialog.callback = callback
+        dialog.selectedWallet = selectedWallet
+        dialog.selectedAccount = selectedAccount
+        dialog.modalPresentationStyle = .pageSheet
+        vc.present(dialog, animated: true, completion: nil)
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        self.titleLabel.text = self.dialogTitle
+
+        self.walletsTableView.hideEmptyAndExtraRows()
+        self.walletsTableView.dataSource = self
+        self.walletsTableView.delegate = self
+        
+        //Without these two lines the grouped UITableView has unnecessary top and bottom padding
+        self.walletsTableView.tableHeaderView = UIView(frame: CGRect(x: 0, y: 0, width: 1, height: 0.1))
+        self.walletsTableView.tableFooterView = UIView(frame: CGRect(x: 0, y: 0, width: 1, height: 0.1))
+        self.walletsTableView.registerCellNib(AccountSelectorTableViewCell.self)
+
+        let accountsFilterFn: (DcrlibwalletAccount) -> Bool = { $0.totalBalance > 0 || $0.name != "imported" }
+        self.wallets = WalletLoader.shared.wallets.map({ Wallet.init($0, accountsFilterFn: accountsFilterFn) })
+        self.walletsTableView.reloadData()
+        
+        let tableContentHeight = self.wallets.reduce(0) { cummulativeContentHeight, wallet in
+            return cummulativeContentHeight + self.walletHeaderSectionHeight
+                + CGFloat(wallet.accounts.count) * self.accountCellRowHeight
+        }
+        self.walletsTableViewHeightConstraint.constant = min(
+            tableContentHeight,
+            UIScreen.main.bounds.height * 0.33 // max height = 1/3rd of screen height
+        )
```

Comment:
As a follow up of the previous comment, you'd expect that if you set the height contraint value to `tableContentHeight`, you wouldn't see a scrollbar. But it appears!

I think we'll be better served with the `SelfSizingTable` idea from your other PR. Then set the maximum height to 1/3rd of the screen height. From my testing, this `tableContentHeight` is 20px smaller than the value of `tableView.contentSize.height`.